sympy/series/__init__.py

```diff
@@ -9,10 +9,12 @@
                         SeqMul)
 from .fourier import fourier_series
 from .formal import fps
-from .limitseq import difference_delta
+from .limitseq import difference_delta, limit_seq
 
 O = Order
+dd = difference_delta
```

Comment:
I'd rather this wasn't shortened (short functions/variables may clash with user things).